### PR TITLE
Ang 4 moving files to app and content + trimming load + PWA enhancements

### PR DESCRIPTION
**Pull request by sendilkumarn**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

**Comment by sendilkumarn** [comment]
@deepu105 FYI

File: generators/client/templates/angular/_package.json [hunk 1]
```diff
@@ -157,7 +157,7 @@
     "e2e": "protractor <%= TEST_SRC_DIR %>protractor.conf.js",
     "postinstall": "webdriver-manager update && node node_modules/phantomjs-prebuilt/install.js && <%= clientPackageManager %> run webpack:build"
     <%_ } else { _%>
-    "postinstall": "node node_modules/phantomjs-prebuilt/install.js && <%= clientPackageManager %> run webpack:build"
+    "postinstall": "node node_modules/phantomjs-prebuilt/install.js"
```
Comment (deepu105) [inline generators/client/templates/angular/_package.json]: why removed?

**Comment by sendilkumarn** [comment]
@deepu105 same here. we can merge this

File: generators/client/templates/angular/_package.json (hunk 1 above)
Comment (deepu105) [inline generators/client/templates/angular/_package.json]: @sendilkumarn you didnt say why this is removed?

Comment (sendilkumarn) [inline generators/client/templates/angular/_package.json]: @deepu105 will merge this. This will make build to run only once

**Comment by sendilkumarn** [comment]
`angular 1`

ng1


`angular2`

ng2



After we merge this PR, this is where we will end up.

For Best Practices, it is more about HTTP2 push.

We have to work on pay load reduction in ng2 for sure and also make sure both ng1 and ng2 are reduce the load time.

File: generators/client/templates/angular/_package.json (hunk 1 above)
Comment (deepu105) [inline generators/client/templates/angular/_package.json]: Yes but if someone tries to launch app without running yarn start then it would not work

Comment (deepu105) [inline generators/client/templates/angular/_package.json]: Yes but if someone tries to launch app without running yarn start then it would not work

Comment (deepu105) [inline generators/client/templates/angular/_package.json]: I guess best solution would be to check if build is already run and avoid running bit again from script. Like how its done for vendor script

**Comment by mraible** [comment]
@sendilkumarn Can you attach a PDF of the report or screenshot so we can see what's missing to get to 100 with Angular?

**Comment by sendilkumarn** [comment]
@mraible 

[Angular1.pdf](https://github.com/jhipster/generator-jhipster/files/1062669/Angular1.pdf)
[Angular2.pdf](https://github.com/jhipster/generator-jhipster/files/1062670/Angular2.pdf)

**Pull request merged**